pkg/controller/nstemplateset/nstemplateset_controller_test.go

```diff
@@ -706,37 +716,40 @@ func newClusterResourceQuota(username, tier string) *quotav1.ClusterResourceQuot
 	}
 }
 
-func getTemplateContent(decoder runtime.Decoder) func(tierName, typeName string) (*templatev1.Template, error) {
-	return func(tierName, typeName string) (*templatev1.Template, error) {
+func getTemplateContent(decoder runtime.Decoder) func(templateRef string) (*tierTemplate, error) {
+	return func(templateRef string) (*tierTemplate, error) {
+		refParts := strings.Split(templateRef, "-")
+		tierName := refParts[0]
+		typeName := refParts[1]
 		if typeName == "fail" || tierName == "fail" {
 			return nil, fmt.Errorf("failed to retrieve template")
 		}
 		var tmplContent string
 		switch tierName {
 		case "advanced": // assume that this tier has a "cluster resources" template
 			switch typeName {
-			case ClusterResources:
+			case "cluster":
```

Comment:
we use `clusterresources` for the templateRef value. Should we use the same value here for the sake of consistency?

Reply:
wait, but in host operator there are files named `cluster.yaml` so I thought that the name should be the same - it would be good to unify it so we avoid such confusion.
What is the proper naming then?

Reply:
> in host operator there are files named cluster.yaml so I thought that the name should be the same

that's a fair point. But we also don't have strictly the same file names for the namespace templates, AFAICS: https://github.com/codeready-toolchain/host-operator/tree/master/deploy/templates/nstemplatetiers/advanced

Reply:
the namespace files look to have pretty strict naming:
`ns_<type>.yaml`
thus I would expect that the resource name of the TierTemplate containing cluster resources should use the file name `cluster`, not something else. 

To avoid confusion in the future I would propose to rename the files (or change the TierTemplate resource naming but this seems to be more complicated.)

Reply:
changed in e61459d